requirements.txt

```diff
@@ -2,18 +2,16 @@ argh==0.26.2
 ipdb==0.12.3
 jsonlines==1.2.0
 marisa_trie==0.7.5
-matplotlib==3.1.1
 mock==3.0.5
 networkx==2.4
 nltk==3.4.5
 notebook==6.0.3
 numpy==1.17.3
 pandas==0.25.3
 pytest==5.3.0
-seaborn==0.9.0
 scikit_learn==0.22
-scipy==1.3.1
 spacy==2.3.5
+scipy~=1.3.1
```

Comment:
Not gonna lie. Didn't even know you could do this. Outside of pytorch and transformers, most versions are not strict. I'm thinking of migrating to something called https://python-poetry.org/. A friend recommended it. We'll relax these requirements in future releases.

Reply:
Interesting! I'll take a look at its documentation.